src/types.ts

```diff
@@ -46,6 +46,7 @@ export type CollapsibleProps<T extends ParamList> = {
   initialTabName?: T
   containerRef: React.RefObject<ContainerRef>
   headerHeight?: number
+  collapsedHeaderHeight?: number
```

Comment:
Nice work 👍  I tested and everything seems to be working. Just one thing, I think `minHeaderHeight` makes more sense instead of `collapsedHeaderHeight` since we are passing the min value of the header. Can you make this little change, please? Also, it needs to be rebased, now the documentation is auto-generated from the types, see the latest readme.